PetaPoco.Tests.Integration/Documentation/Delete.cs

```diff
@@ -20,7 +20,10 @@ public Deletes()
         [Fact]
         public void DeleteByPoco()
         {
-            // Create the person
+            // Clear out any people and reset the ID sequence counter
+            DB.Execute("TRUNCATE TABLE [People]");
```

Comment:
I thought each test had a fresh DB instance?

Reply:
They do. If you look at the build scripts, they should all reset the DB and then build it back up again. Ref https://github.com/CollaboratingPlatypus/PetaPoco/blob/development/PetaPoco.Tests.Integration/Scripts/MySqlBuildDatabase.sql

Reply:
My reasoning for adding that was due to the fact that 1) the documentation tests are used (in the docs) independent of the test suite, and 2) some of the tests *did* include that reset. Since the assertions require an empty table each run to pass anyways, and as they're presented outside the context of the actual test project, the added consistency seemed justifiable.

Reply:
Oh, I see. I am fairly sure the docs tests don't need to `truncate`. Could you try removing it? The idea is that all integration tests start with a fresh database, and how that's done is left to the database script. This is important because we don't have/shouldn't ever need to place vendor-specific SQL in the base tests.

Reply:
Having consistency between the documentation tests was my main concern here, but I do agree with you they should be removed, and will include it in the next commit. (Note the discrepency between [`DeleteByPoco()` and `DeleteByPrimaryKey()`](https://github.com/CollaboratingPlatypus/PetaPoco/wiki/Deleting#examples) in the wiki).

Since the `TRUNCATE` commands were in *some* but not all of the doc tests for Insert/Delete/Update/Save previously, as in the link above, you want those removed also I presume? I'm pretty sure they are only there as artifacts from the early days prior to integrating XUnit.

Reply:
Sorry. Yes please, @Ste1io. 

> I'm pretty sure they are only there as artifacts from the early days prior to integrating XUnit.

I wish that were the case, but it's probably more likely it was, ehm, me. Early on, after I took over from Brad, there was little in the way of code reviews :)